shortnames.conf

```diff
@@ -34,10 +34,10 @@
   "leap-microdnf" = "registry.opensuse.org/opensuse/leap-microdnf"
   "tw-busybox" = "registry.opensuse.org/opensuse/busybox"
   # SUSE
-  "suse/sle15" = "registry.suse.com/suse/sle15"
-  "suse/sles12sp5" = "registry.suse.com/suse/sles12sp5"
-  "suse/sles12sp4" = "registry.suse.com/suse/sles12sp4"
-  "suse/sles12sp3" = "registry.suse.com/suse/sles12sp3"
+  "bci/bci-base" = "registry.suse.com/bci/bci-base"
+  "bci/bci-micro" = "registry.suse.com/bci/bci-micro"
+  "bci/bci-minimal" = "registry.suse.com/bci/bci-minimal"
+  "bci/bci-busybox" = "registry.suse.com/bci/bci-busybox"
```

Comment:
Just to make sure: Are you sure it's safe to remove the SLES aliases?  Some users may very well use them and it would break on update.

I'd feel safer by keep the four SLES aliases and add BCI on top.  But I leave the decision up to you.

Reply:
IMO the previous aliases should be kept - they won't hurt and are definitely still used.

Reply:
You're right, it's probably not a good idea to remove them.

Reply:
We should definitely keep them, for when BCI one day moves to be based on another product. 
Thanks Valentin for your input.

Reply:
Nice to see you here, Alex :)